sdk/identity/azure-identity/azure/identity/_credentials/default.py

```diff
@@ -194,7 +194,9 @@ def __init__(self, **kwargs: Any) -> None:  # pylint: disable=too-many-statement
 
         super(DefaultAzureCredential, self).__init__(*credentials)
 
-    def get_token(self, *scopes: str, **kwargs) -> AccessToken:
+    def get_token(
+        self, *scopes: str, claims: Optional[str] = None, tenant_id: Optional[str] = None, **kwargs: Any
```

Comment:
This will remove `claims` & `tenant_id` from `kwargs`.

You need to double check if this changes behaviors.

Reply:
Updated to use a helper method that helps us update `kwargs` when parameters are provided. e.g. https://github.com/Azure/azure-sdk-for-python/blob/727565fa49f5790b877457a1a6b7150c2224338f/sdk/identity/azure-identity/azure/identity/_internal/get_token_mixin.py#L82-L83